platform/qt/src/string_stdlib.cpp

```diff
@@ -9,13 +9,13 @@ namespace mbgl {
 namespace platform {
 
 std::string uppercase(const std::string& str) {
```

Comment:
@brunoabinader I added this change after you reviewed. Do you mind having a look again?